src/types.jl

```diff
@@ -102,6 +103,11 @@ Base.@kwdef struct Sets
     # Segments
     num_segments::Int64
     segment_idx::UnitRange{Int64}
+    ## Demand Flexibility
+    flexible_bus_idx_up::Union{Array{Int64,1}, Nothing}
+    flexible_bus_idx_dn::Union{Array{Int64,1}, Nothing}
+    num_flexible_bus_up::Int64
+    num_flexible_bus_dn::Int64
```

Comment:
Since we are assuming the buses with demand flexibility have both an up and a down component, should we have only have one `flexible_bus_idx` and one `num_flexible_bus`?

Reply:
yes if that is what's desired.. One of my questions was asking if the assumption should be made or not. It looks like it is the way to go and I will remove it soon.